src/prefixmaps/data/merged.csv

```diff
@@ -1,4297 +1,4299 @@
-merged,vss,http://automotive.eurecom.fr/vsso#,namespace_alias
-merged,vsso,http://automotive.eurecom.fr/vsso#,canonical
-merged,vsto,http://escience.rpi.edu/ontology/vsto/2/0/vsto.owl#,canonical
-merged,vstoi,http://hadatac.org/ont/vstoi#,canonical
-merged,vsw,http://verticalsearchworks.com/ontology/,canonical
-merged,vsws,http://verticalsearchworks.com/ontology/synset#,canonical
-merged,VT,http://purl.obolibrary.org/obo/VT_,canonical
-merged,VTO,http://purl.obolibrary.org/obo/VTO_,canonical
-merged,vvo,http://purl.org/vvo/ns#,canonical
-merged,w3cgeo,http://www.w3.org/2003/01/geo/wgs84_pos#,namespace_alias
-merged,w3con,http://www.w3.org/2000/10/swap/pim/contact#,namespace_alias
-merged,w3geo,http://www.w3.org/2003/01/geo/wgs84_pos#,namespace_alias
-merged,w3id,https://w3id.org/,canonical
-merged,w3p,http://prov4j.org/w3p/,canonical
-merged,w3po,http://purl.org/provenance/w3p/w3po#,canonical
-merged,waa,http://purl.oclc.org/NET/WebApiAuthentication#,canonical
-merged,waarde,https://lod.milieuinfo.be/ns/waarde#,canonical
-merged,wab,http://wab.uib.no/cost-a32_philospace/wittgenstein.owl#,canonical
-merged,wai,http://purl.org/wai#,canonical
-merged,wail,http://www.eyrie.org/~zednenem/2002/wail/,canonical
-merged,wairole,http://www.w3.org/2005/01/wai-rdf/GUIRoleTaxonomy#,canonical
-merged,walmart,https://www.amazon.de/,canonical
-merged,wao,http://webtlab.it.uc3m.es/2010/10/WebAppsOntology#,canonical
-merged,wapp,http://ns.rww.io/wapp#,canonical
-merged,wasa,http://vocab.sti2.at/wasa/,canonical
-merged,water,http://escience.rpi.edu/ontology/semanteco/2/0/water.owl#,canonical
-merged,WB,https://identifiers.org/wb/,canonical
-merged,wb,http://data.worldbank.org/,prefix_alias
-merged,WB.RNAI,http://identifiers.org/wb.rnai/,canonical
-merged,WBbt,http://purl.obolibrary.org/obo/WBbt_,canonical
-merged,wbc,http://worldbank.270a.info/classification/,canonical
-merged,WBls,http://purl.obolibrary.org/obo/WBls_,canonical
-merged,wbp,http://worldbank.270a.info/property/,canonical
-merged,WBPhenotype,http://purl.obolibrary.org/obo/WBPhenotype_,canonical
-merged,wd,http://www.wikidata.org/entity/,canonical
-merged,wde,http://www.wikidata.org/entity/,namespace_alias
-merged,wdno,http://www.wikidata.org/prop/novalue/,canonical
-merged,wdr,http://www.w3.org/2007/05/powder#,namespace_alias
-merged,wdrs,http://www.w3.org/2007/05/powder-s#,canonical
-merged,wds,http://www.wikidata.org/entity/statement/,canonical
-merged,wdt,http://www.wikidata.org/prop/direct/,canonical
-merged,wdtn,http://www.wikidata.org/prop/direct-normalized/,canonical
-merged,wdv,http://www.wikidata.org/value/,canonical
-merged,web,http://www.w3.org/2007/uwa/context/web.owl#,canonical
-merged,webac,http://fedora.info/definitions/v4/webac#,canonical
-merged,webbox,http://webbox.ecs.soton.ac.uk/ns#,canonical
-merged,WEBELEMENTS,https://www.webelements.com/,canonical
-merged,webservice,http://www.openlinksw.com/ontology/webservices#,canonical
-merged,webtlab,http://webtlab.it.uc3m.es/,can
```

Comment:
See here - `WIKIDATA` comes from the Bioregistry

Reply:
The prefix in bioregistry is `wikidata`, not `WIKIDATA` right?

Reply:
correct, but the processing in this repo uppercases the bioregistry prefixes

Reply:
right; so should the source be prefixmaps?

Reply:
I don't consider prefixmaps to be its own registry, this is just a transform on a Bioregistry prefix. Given #48 and https://github.com/biopragmatics/bioregistry/pull/969, it should be more obvious how all uppercase prefix synonyms get in

Reply:
@sierra-moxon so to try and explain again, the point of the source column is to help track down where a given expansion came from. In this case, even though a transformation was done on it, it's still from the Bioregistry.

Reply:
how about we call it "expansion_source" ?

Reply:
@cthoyt - did you see that this was "approved" and had this question for you?  Please merge as you see fit, but consider being more explicit in the naming of this column.

Reply:
I did not see this, thank you for the ping. I will update that as you suggested :)